lib/commands/project.ts

```diff
@@ -28,10 +31,125 @@ export class ProjectCommandBase implements ICommand {
 	}
 }
 
+export class InitProjectCommandBase extends ProjectCommandBase {
+	protected projectDir: string;
+	protected tnsModulesDir: string;
+	protected bootstrapFile: string;
+	protected indexHtml: string;
+
+	constructor(protected $project: Project.IProject,
+				protected $errors: IErrors,
+				protected $fs: IFileSystem,
+				protected $logger: ILogger,
+				protected $projectNameValidator?: IProjectNameValidator) {
+		super($project, $errors, $projectNameValidator);
+
+		this.projectDir = this.$project.getNewProjectDir();
+		this.tnsModulesDir = path.join(this.projectDir, "tns_modules");
+		this.bootstrapFile = path.join(this.projectDir, "app", "bootstrap.js");
+		this.indexHtml = path.join(this.projectDir, "index.html");
+	}
+
+	public execute(args: string[]): IFuture<void> {
+		return null;
+	}
```

Comment:
This function should not execute ever, right? In this case, I think it is better to write a ```throw Error(Never execute, write us a support ticket)```